pkg/services/ngalert/api/api.go

```diff
@@ -130,14 +129,9 @@ func (api *API) conditionEvalEndpoint(c *models.ReqContext, cmd ngmodels.EvalAle
 	}
 
 	frame := evalResults.AsDataFrame()
-	df := plugins.NewDecodedDataFrames([]*data.Frame{&frame})
-	instances, err := df.Encoded()
-	if err != nil {
-		return response.Error(400, "Failed to encode result dataframes", err)
-	}
 
-	return response.JSON(200, util.DynMap{
-		"instances": instances,
+	return response.JSONStreaming(200, util.DynMap{
+		"instances": []*data.Frame{&frame},
```

Comment:
does this need/want to be an array?  in ever case it looks like it is just a single instance